proto/kvrpcpb.proto

```diff
@@ -863,6 +863,8 @@ message TimeDetail {
     // cannot be excluded for now, like Mutex wait time, which is included in this field, so that
     // this field is called wall time instead of CPU time.
     int64 process_wall_time_ms = 2;
+    // Scan Used Time means the time used in TiKV scan.
+    int64 scan_used_time_ms =3;
```

Comment:
```suggestion
    int64 kv_read_wall_time_ms = 3;
```

Reply:
I think calling it read is better, since read operation includes get and scan, and actually we want time measurements for both.